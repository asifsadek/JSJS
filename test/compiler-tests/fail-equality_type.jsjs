val print_length = /\(l: list num) => {
  print(List.length(l));
};

val res = if (print_length([1,2,3,4]) == 4) then "nope!!" else "yup!!";

print(res);
