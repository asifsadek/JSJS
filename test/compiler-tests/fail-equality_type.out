Type error: expected value of type 'num', got a value of type 'unit' instead
